SpatialGDK/Source/SpatialGDK/Public/SpatialGDKSettings.h

```diff
@@ -123,6 +123,10 @@ class SPATIALGDK_API USpatialGDKSettings : public UObject
 	UPROPERTY(config, meta = (ConfigRestartRequired = false))
 	bool bCheckRPCOrder;
 
+	/** Seconds to wait before executing an RPC substituting null values for unresoled parameters */
+	UPROPERTY(config, meta = (ConfigRestartRequired = false))
```

Comment:
I think we should include this in the visible GDK settings under the category "Replication"

Reply:
Added.